<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChosenBackgroundColor</class>
 <widget class="QWidget" name="ChosenBackgroundColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChosenBackgroundColor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="dialogButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Choose color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="colorPreview">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
